app/presenters/candidate_interface/application_form_presenter.rb

```diff
@@ -121,50 +131,48 @@ def ready_to_submit?
     end
 
     def application_choices_added?
-      @application_form.application_choices.present?
+      application_form.application_choices.present?
     end
 
-    def personal_details_completed?
-      @application_form.personal_details_completed?
-    end
+    delegate :personal_details_completed?, to: :application_form
```

Comment:
Is there a reason this is here and not at the top with the rest of the delegation? (actually wondering)

Reply:
Ah did not see that - will change. Also need to add some specs for the ApplicationFormPresenter

Reply:
Done